src/v2/guide/index.md

```diff
@@ -393,6 +393,8 @@ You may have noticed that Vue components are very similar to **Custom Elements**
 
 2. Vue components provide important features that are not available in plain custom elements, most notably cross-component data flow, custom event communication and build tool integrations.
 
+According to <https://custom-elements-everywhere.com/#vue>, Vue has a great interoperability with **Custom Elements**.
```

Comment:
We can provide a bit more context:

"Although Vue doesn't use custom elements internally, it has great interoperability when it comes to consuming or distributing as custom elements. [According to ...]. Vue CLI also supports building Vue components that register themselves as native custom elements."